### ISSUE TITLE
losses aggregated incorrecly when training multi-output models via model.fit()

### ISSUE THREAD

**Issue by vector-space** [keras-team/keras#19337]
My model has 7 outputs and my tfdataset yields x, y.
In keras 2.x I just need to specify a loss, and model.fit() will apply loss_fn(y, p1), loss_fn(y, p2), loss_fn(y, p3)...separately, then sum the losses together and show loss:xx, loss1:xx, loss2:xx, loss3:xx...on the console, everything is good.
But in keras 3.x these behavier has changed, model.fit() will now apply loss_fn only on the first output of the model.
And even I modified my tfdataset, force it to yield x, (y,) * 7, thers is still one loss shown on console, the value seems to be the sum of all losses.
Specifying metrics like metrics={'main': ['accuracy', 'AUC', mean_iou], 'lv1':None, 'lv2':None, 'lv3':None, ...} doesn't make things better.
I want to see the summed loss as well as every separate loss on the console and I don't want to change my data generator output to (x, (y, y, y, y, y, y, y)), that looks clumsy. What should I do? Or could keras change the fit() behavier back just like in 2.x?
My env is tf2.16.1, keras3.0.5, python3.11.

**Comment by sachinprasadhs** [comment]
Hi, Please refer the comment [here](https://github.com/keras-team/keras/issues/19159#issuecomment-1947138497) and let us know if this helps you.

**Comment by vector-space** [comment]
> Hi, Please refer the comment [here](https://github.com/keras-team/keras/issues/19159#issuecomment-1947138497) and let us know if this helps you.

Well, I got to know that this change is on some other purpose and I decide to write a custom training loop(just like using torch) to solve my problem. Anyway, thanks for your response^-^

**Comment by innat** [comment]
https://github.com/keras-team/keras/issues/19219

**Comment by sachinprasadhs** [comment]
@vector-space , You can close this issue and track the progress in the linked issue.
